docs/setup/discovery-server.md

```diff
@@ -0,0 +1,46 @@
+# Fast DDS Discovery Server
+
+This page gives instructions on how to use a Fast DDS discovery server with the Create® 3 robot as a client.
+For more information on the Fast DDS discovery server, please refer to the [eProsima documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/ros2/discovery_server/ros2_discovery_server.html).
+
+*We’ll use the term "computer" to refer to any SBC (single board computer), virtual machine, personal computer, etc.*
+
+## Configure Server
+
+1. Start the discovery server on the computer that you wish to use as your server by replacing <SERVER_IP> with the server’s IP address.
+
+        fastdds discovery -i 0 -l <SERVER_IP> -p 11811
+   
+     ![](./data/fastdds.png)
+
+## Configure Create® 3 Robot as a Client
+!!! important
+    If you have enabled the discovery server previously, make sure it is disabled in the configuration settings **before** connecting it to a new Wi-Fi network.*
+
+1. Make sure your Create® 3 robot is connected to Wi-Fi. If it isn't, follow the directions [here](../provision/).
+
+1. Navigate to the robot’s web server via its IP address.
+Go to [webserver application settings](../../webserver/application/) and enable the checkbox for the discovery server.
+In the field `Address and port of Fast DDS discovery server`, enter the IP address of your server followed by `:11811` (the default port for the Fast DDS discovery server is `11811`, but it is configurable).
+
+1. If you intend on connecting multiple robots to the same discovery server, make sure to give the Create® 3 robot a namespace.
+Restart the application after saving all settings.
+  ![](./data/app-config.png)
+
+!!! attention
+      It is recommended you check the [logs](../../webserver/logs/) to confirm the discovery server has been enabled on the robot. ![](./data/logs.png)
+
+## Configure Other Devices as Super Clients 
+When using a discovery server with a Create® 3 Robot, all other devices connected to the discovery server must be set up as super clients in order to communicate with the Create® 3 Robot.
```

Comment:
We should clarify what are the "other devices" mentioned here.

Reply:
My understanding is this could be any device running ROS 2, correct?